### DISCUSSION TITLE
Request Lock the Dock

### DISCUSSION THREAD

**Discussion by billouetaudrey** [linuxdeepin/developer-center#2909] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello, is it possible to add settings for Lock the dock (lock rearrange icon etc...)

Thanks 🙏

**Comment by meiyixiang** [comment]
You don't understand exactly what you want, can you take a screenshot and add a description?

**Comment by billouetaudrey** [comment]
Hello, I just want to lock the dock so that I don't move my icons by mistake ( shortcut icon in center ) 

你好，我只是想挡住dock，以免误动我的图标

**Comment by meiyixiang** [comment]
I understand what you mean. I have fed back your suggestions to the relevant product manager

**Comment by billouetaudrey** [comment]
Thanks :) 

Good Luck :)
